docs/tutorials/how-tos/use-tokens/earn-passive-income-as-a-liquidity-provider.md

```diff
@@ -0,0 +1,227 @@
+---
+parent: use-tokens.html
+seo:
+    description: Provide liquidity for an Automated Market Maker (AMM) and earn income from trading fees.
+embed_xrpl_js: true
+status: not_enabled
+filters:
+  - interactive_steps
```

Comment:
The `embed_xrpl_js` and `filters` frontmatter elements are vestiges of the old Dactyl-based site and can be omitted here. (Same for `parent` actually.)